test/training.spec.js

```diff
@@ -3,20 +3,32 @@ const utils = require('./helpers/utils'),
   {
     basicFiles,
     sequelizeFiles,
+    sequelizeFilesGraphql,
     travisFiles,
     herokuFiles,
     jenkisFiles,
     dockerFiles,
-    examplePrompts
+    examplePrompts,
+    basicFilesGraphql
   } = require('./helpers/constants');
 
-describe('WTraining project', () => {
+const testSnapshot = technology => () => {
   beforeAll(() => {
     mockCommand();
-    return utils.runKickoff({ ...examplePrompts, inTraining: true });
+    return utils.runKickoff({ ...examplePrompts, inTraining: true, technology });
   });
   test('creates training files', () => {
+    if (technology === 'graphQL') {
+      utils.checkExistentFiles(
+        [basicFilesGraphql, sequelizeFilesGraphql, travisFiles, herokuFiles],
+        'WTraining'
+      );
+      return utils.checkNonExistentFiles([jenkisFiles, dockerFiles], 'WTraining');
+    }
     utils.checkExistentFiles([basicFiles, sequelizeFiles, travisFiles, herokuFiles], 'WTraining');
-    utils.checkNonExistentFiles([jenkisFiles, dockerFiles], 'WTraining');
+    return utils.checkNonExistentFiles([jenkisFiles, dockerFiles], 'WTraining');
   });
-});
+};
+
+describe('WTraining project', testSnapshot('nodeJS'));
+describe('WTraining project', testSnapshot('graphQL'));
```

Comment:
Instead of repeat in each test suite both techs, could be defined as constants and perform a `describe.each` with that

Reply:
ok!